README.md

```diff
@@ -7,7 +7,74 @@
 
 Python Library for Evaluation
 
-## MT-Bench / MT-Bench-Branch Testing Steps
+## What is Evaluation?
+
+Evaluation is the step that allows us to gauge how a given model is performing against a set of specific tasks, by running known and standardized benchmark tests against
+the model. Running this step gives us numerical scores across these various benchmarks, as well as logged excerpts/samples of the outputs the model produced during these
+benchmarks. Using a combination of these artifacts as reference, along with manual smoke screening of a model allows us to get the best idea about whether or not a model
+has learned and improved on something we are trying to teach it. There are 2 stages in the InstructLab process where we perform model evaluation:
+
+### Inter-checkpoint Evaluation
+
+This step occurs during multi-phase training. Each phase of training produces a number of different “checkpoints” of the model that are taken at various stages during
+the phase. At the end of each phase, we need to evaluate all the checkpoints in order to find the one that provides the best results. This is done as part of the
+[InstructLab Training](https://github.com/instructlab/training) library.
+
+### Full-scale final Evaluation
+
+Once training is complete, and we have picked the best checkpoint from the output of the final phase, we can run full-scale evaluation suite which runs MT-Bench, MMLU,
+MT-Bench Branch and MMLU Branch.
+
+## Methods of Evaluation
+
+Below are more in-depth explanations of the suite of benchmarks we are using as methods for evaluation of models.
+
+### Multi-turn benchmark (MT-Bench)
+
+**tl;dr** Full model evaluation of performance on **skills**
+
+MT-Bench is a type of benchmarking that involves asking a model 80 multi-turn questions - i.e.
+
+```text
+<Question 1> → <model’s answer 1> → <Follow-up question> → <model’s answer 2>
+```
+
+and having a “judge” model review the given multi-turn question, the provided model answer, and rate the answer with a score out of 10. The scores are then averaged out
+and the final score produced is the “MT-bench score” for that model. This benchmark assumes no factual knowledge on the model’s part. The questions are static, but do
+not get obsolete with time.
+
+You can read more about MT-Bench [here](https://arxiv.org/abs/2306.05685)
+
+### MT-Bench Branch
+
+MT-Bench Branch is an adaptation of MT-Bench that is designed to test custom skills that are added/being added to the model via the InstructLab project. These new skills
+come in the form of question/answer pairs in a Git branch of the [taxonomy](https://github.com/instructlab/taxonomy).
+
+MT-Bench Branch uses the user supplied seed questions to have the candidate model generate answers to, which are then judged by the judge model.
```

Comment:
Should we refer to using the answers from the qnas as providing the reference answers here?

Reply:
Added a note on this, let me know what you think!